### ISSUE TITLE
Latest version not showing menu

### ISSUE THREAD

**Issue by cymplecy** [sakazuki/node-red-desktop#60]
![image](https://user-images.githubusercontent.com/2357428/145693369-bfd42ece-3f2c-41a7-9432-ba969d612197.png)

previous version
![image](https://user-images.githubusercontent.com/2357428/145693357-f8271342-42b5-4403-ab3b-91ba9bfa85fc.png)

**Comment by ceysa75** [comment]
You are right. Menu is not shown by default. Just push the ALT button on your keyboard.

Regards

Ceysa
